srcpkgs/perl-File-MimeInfo/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'perl-File-MimeInfo'
 pkgname=perl-File-MimeInfo
-version=0.29
-revision=2
+version=0.30
+revision=1
 wrksrc="${pkgname/perl-/}-${version}"
 build_style=perl-module
 hostmakedepends="perl"
-makedepends="${hostmakedepends} perl-File-BaseDir perl-File-DesktopEntry"
+makedepends="${hostmakedepends} perl-File-BaseDir perl-File-DesktopEntry perl-Encode-Locale"
```

Comment:
Please drop `${hostmakedepends}` from `makedepends`. The other `perl-*` packages will automatically pull in `perl` itself, plus using `$hostmakedepends` like that is bad form since a later change to `hostmakedepends` might cause unnoticed `makedepends` creep.

Reply:
Done.